tests/Behat/Context/Reorder/ReorderContext.php

```diff
@@ -130,4 +137,23 @@ public function iProceedToThePaymentStep(): void
     {
 
     }
+
+    private function assertFlashMessageWithTextExists(string $text) {
+        $notifications = $this->session->getPage()->findAll('css', '.sylius-flash-message');
+
+        if (null === $notifications) {
+            throw new \Exception('There is no notification on current page.');
+        }
+
+        /** @var NodeElement $notification */
+        foreach ($notifications as $notification) {
+            $message = $notification->getText();
+
+            if (strpos($message, $text)) {
+                return;
+            }
+        }
+
+        throw new \Exception(sprintf('Flash message with text %s not found', $text));
```

Comment:
You could consider using `Sylius\Behat\Service\NotificationChecker`

Reply:
Unfortunately, since flash messages used in this PR have info level and NotificationChecker is able to distinguish only success and failure messages, checking flashes' behaviour would require additional Sylius contribution.